### PR TITLE
ci: add capture HA test

### LINKED ISSUE #408: Ensure tables are reblanced when a capture is killed
# Cases

- [x] Kill a capture
- [x] Hang up a capture until the session expires
- [x] Hang up the capture and then resume it

# Expectation
* The tables are rebalanced
* The capture suicides and recovers when its session expires

### PR DESCRIPTION
**Pull request by shafreeck**
This is a PR based on #420

### What problem does this PR solve? 

Add test script for capture HA test

Closes #408

### CODE REVIEW

File: tests/availability/capture.sh [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    # wait for the tables to appear
+    check_table_exists test.availability ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 20
+
+    run_sql "INSERT INTO test.availability(id, val) VALUES (1, 1);"
+    ensure $MAX_RETRIES nonempty 'select id, val from test.availability where id=1 and val=1'
+
+    # start the second capture
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    capture_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}' | grep -v "$owner_id")
```
Comment (amyangfei) [inline tests/availability/capture.sh:68]: recommend to add some retry mechanism here

File: tests/availability/capture.sh [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    # ensure the server become the owner
+    ensure $MAX_RETRIES "$CDC_BINARY cli capture list 2>&1 | grep '\"is-owner\": true'"
+    owner_pid=$(pidof $CDC_BINARY)
+    owner_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}')
+    echo "owner pid:" $owner_pid
+    echo "owner id" $owner_id
+
+    # start the second capture
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    capture_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}' | grep -v "$owner_id")
```
Comment (amyangfei) [inline tests/availability/capture.sh:97]: ditto

**Comment by leoppro** [comment]
LGTM

**Pull request merged**